video.md

```diff
@@ -2,26 +2,19 @@
 
 # CodeRefinery: welcome and practical information for workshops
 
-## Arrival
-
-Please test the following when arriving:
-
-- Test that you have Git installed: `git --version`
-- Check [Git configuration instructions](https://coderefinery.github.io/git-refresher/01-setup/#configuring-git).
-
 
 ## Welcome
 
+- Presentation of instructors and helpers.
 - Software Carpentry style of teaching: code-along and many exercises, minimize slides.
 - Instructor gives a [tour of Zoom](https://github.com/coderefinery/manuals/blob/master/zoom-mechanics.md)
 - Announce that Zoom chat can be used for questions but that the content will
   be deleted afterwards.
+- We clarify to participants whether and how we record and archive and inform participants
+  how they can participate while maintaining privacy for them and others.
 - We will use a HackMD for questions/notes/links that we want to keep.
-- We will neither record the session, nor archive the chat.
-- Presentation of instructors and helpers.
 - Instructor gives a tour of HackMD which contains a list of participants' initials.
-- Ice-breaker: ask participants to use their real name on Zoom,
-  roll-call for participants to write their name in the HackMD,
+- Ice-breaker: roll-call for participants to write their name in the HackMD,
   their experience with Git, and their operating system (we will use this information to form groups).
```

Comment:
What about a unrelated  fun question, such as "some tip or trick that you think would benefit someone"?

Not sure if it should be added unless other things are removed, since otherwise there might be too much much to do at once (or maybe not?).  Those other things are useful, too, though maybe we don't use the OS check so much these days?

Reply:
Good idea, I will modify this.